src/lib/nodes/eth/EthClient.ts

```diff
@@ -111,12 +114,25 @@ export class EthClient extends Client<EthNode> {
     })
   }
 
+  /**
+   * Get client instance depending on availability of a domain.
+   * @returns { Web3Eth } Web3 Ethereum module instance.
+   */
+  getClientInstance(): Web3Eth {
+    const node = this.getNode()
+
+    return node.client()
+  }
```

Comment:
```suggestion
  getNodeClient(): Web3Eth {
    return this.getNode().client()
  }
```

1. Better name
2. Simplify

Reply:
https://github.com/Adamant-im/adamant-im/pull/768/commits/2e7236e2682939ee302a41cdcfde08985c248d37